bot/controllers/twitchLive.js

```diff
@@ -0,0 +1,32 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+// const util = require('apex-util');
+
+class StreamerLiveController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!twitchLive',
+        '!twitchLive <twitch_name> <twitch_game>',
+        'Twitch Live',
+        'A quick easy way to announce you are going live on Twitch.',
+        this.addTwitchAction.bind(controller),
+        'reply',
+        true,
+      ),
+    ];
+  }
+  addTwitchAction() {
```

Comment:
Consider changing this to "streamerLiveAction" to better fit the overall functionality.

Reply:
changed the method name